include/toml++/impl/preprocessor.h

```diff
@@ -75,14 +75,22 @@
 	#if TOML_CLANG >= 10
 		#define TOML_DISABLE_SPAM_WARNINGS_CLANG_10 \
 			_Pragma("clang diagnostic ignored \"-Wzero-as-null-pointer-constant\"") \
-			_Pragma("clang diagnostic ignored \"-Wsuggest-destructor-override\"") \
 			static_assert(true)
 	#else
 		#define TOML_DISABLE_SPAM_WARNINGS_CLANG_10 static_assert(true)
 	#endif
 
+	#if TOML_CLANG >= 11
+		#define TOML_DISABLE_SPAM_WARNINGS_CLANG_11 \
```

Comment:
An #undef for this macro should also be added to the #undef block at bottom of toml.h so it doesn't leak out of the library.

Reply:
Sure, will fix!

Reply:
Fixed!